pkg/kubelet/certificate/kubelet.go

```diff
@@ -148,3 +153,115 @@ func NewKubeletClientCertificateManager(certDirectory string, nodeName types.Nod
 	}
 	return m, nil
 }
+
+type CertificateUpdater interface {
+	UpdateCertInfo(nodeAddrs []v1.NodeAddress) error
+	//	GenerateCert()
+	Current() *tls.Certificate
+}
+
+type certificateUpdater struct {
+	ipList         []net.IP
+	dnsList        []string
+	host           string
+	needsCertRegen bool
+
+	certMutex sync.RWMutex
+	cert      *tls.Certificate
+}
+
+func NewCertificateUpdater(hostName string, certFile, keyFile string, iplist []net.IP, dnslist []string) (CertificateUpdater, error) {
+	c, err := tls.LoadX509KeyPair(certFile, keyFile)
+	if err != nil {
+		return nil, err
+	}
+	updater := certificateUpdater{
```

Comment:
No need to declare a new variable. Directly return it.

Reply:
Done